### PR TITLE
update benchmark-dockerized.sh and verify-dockerized.sh

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by xichengliudui** [comment]
/assign @fejta, Sorry, I forgot to check your message.
Ref: https://github.com/kubernetes/kubernetes/pull/75915/files#r272434692
https://github.com/kubernetes/kubernetes/pull/75915/files#r272434751

File: hack/jenkins/benchmark-dockerized.sh [hunk 1]
```diff
@@ -57,4 +57,4 @@ cd /go/src/k8s.io/kubernetes
 make test-integration WHAT="$*" KUBE_TEST_ARGS="-run='XXX' -bench=. -benchmem" \
   | tee \
    >(prettybench -no-passthrough > "${ARTIFACTS}/BenchmarkResults.txt") \
-   >(go run test/integration/benchmark/jsonify/main.go "${ARTIFACTS}"/BenchmarkResults_benchmark_"$(date -u +%Y-%m-%dT%H:%M:%SZ)".json || cat > /dev/null)
+   >(go run test/integration/benchmark/jsonify/main.go "${ARTIFACTS}/BenchmarkResults_benchmark_""$(date -u +%Y-%m-%dT%H:%M:%SZ)".json || cat > /dev/null)
```
Comment (cblecker) [inline hack/jenkins/benchmark-dockerized.sh]:
```suggestion
   >(go run test/integration/benchmark/jsonify/main.go "${ARTIFACTS}/BenchmarkResults_benchmark_$(date -u +%Y-%m-%dT%H:%M:%SZ).json" || cat > /dev/null)
```

Comment (xichengliudui) [reply hack/jenkins/benchmark-dockerized.sh]: YES, has been updated.

**Pull request merged**